src/main/java/org/gitlab4j/api/models/CommitRef.java

```diff
@@ -0,0 +1,36 @@
+package org.gitlab4j.api.models;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * @author Евгений Уткин (evgeny.utkin@mediascope.net)
+ */
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class CommitRef {
+
+    private RefType type;
+    private String name;
+
+   public enum RefType {
+        branch, tag, all
+    }
+
```

Comment:
All enums should be uppercase, and to make the enum work correctly with this library this enum should look like this:
```
public enum RefType {

    BRANCH, TAG, ALL;

    private static JacksonJsonEnumHelper<RefType> enumHelper = new JacksonJsonEnumHelper<>(RefType.class);

    @JsonCreator
    public static RefType forValue(String value) {
        return enumHelper.forValue(value);
    }

    @JsonValue
    public String toValue() {
        return (enumHelper.toString(this));
    }

    @Override
    public String toString() {
        return (enumHelper.toString(this));
    }
}
```